packages/neuron-wallet/tests/block-sync-renderer/queue.test.ts

```diff
@@ -101,7 +100,7 @@ describe('queue', () => {
   const fakeWalletId = 'w1'
   const addressInfo: Address = {
     address,
-    blake160: '0xfakeblake160',
+    blake160: `0x${'0'.repeat(40)}`,
```

Comment:
This `blake160` here will be used to calculate `args` then `utils.computeScriptHash(script: Script)`, so that `blake160 ` needs to be a valid `HexString` here, otherwise it won't build.